openedx/features/smart_referral/tests/test_tasks.py

```diff
@@ -4,13 +4,19 @@
 
 from common.lib.mandrill_client.client import MandrillClient
 
-from openedx.features.smart_referral.tasks import task_send_referral_and_toolkit_emails
```

Comment:
run isort or manually remove line 6.

Reply:
I did run isort for sorting imports. If the line break is concerning than we are following a practice to add line break between python, django and project imports. This practice is following in many occurrences  with our project.

Reply:
I agree, but aren't these both project imports? And when I run isort in this file, inside lms-shell, it removes this line break.

Reply:
@muak50